### PR TITLE
Rot 266 contrast

### PR DESCRIPTION
**Pull request by hannah-whelan**
Updated titan's dark pink colour to fix accessibility contrast issue

### CODE REVIEW

**Review by ALougherANDDigital — APPROVED** [review]

File: src/components/app_header/header/HomePageHeader.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const HomepageHeader = () => {
             <br />
             <button
               type="button"
-              className="font-bold bg-titansBrightPink text-white font-bold py-2 px-8 rounded-full"
+              className="bg-titansBrightPink text-white font-bold py-2 px-8 rounded-full"
```
Comment (tom-downes-and) [inline src/components/app_header/header/HomePageHeader.tsx:44]: why change `font-bold`?

**Review by tom-downes-and — COMMENTED** [review]
What has changed with the logo?

File: src/components/app_header/header/HomePageHeader.tsx (hunk 1 above)
Comment (tom-downes-and) [reply src/components/app_header/header/HomePageHeader.tsx:44]: Ah I see its just the order of classes that has changed

Comment (hannah-whelan) [reply src/components/app_header/header/HomePageHeader.tsx:44]: font-bold was in there twice for some reason

Comment (hannah-whelan) [reply src/components/app_header/header/HomePageHeader.tsx:44]: Logo was just the shade of pink changing in the SVG - Zoya will update on figma also

**Pull request merged**